scripts/deploy-mcd

```diff
@@ -10,38 +10,21 @@ function sed_inplace {
   sed -i.bak "$1" $2 && rm $2.bak
 }
 
-function deploy_mcd_tdds {
-  TDDS=$LIB/testchain-dss-deployment-scripts
-  cd $TDDS
+function deploy_mcd_dds {
+  DDS=$LIB/dss-deploy-scripts
+  cd $DDS
 
   export MCD_PAUSE_DELAY=0
 
-  # change price feeds to DSValue so we can set prices from within tests
   TMP=$(mktemp)
-  jq_inplace '.tokens[].pip.type = "value"' step-1.json
 
-  # change step-1-deploy not to send 10000 ETH
-  sed_inplace 's/10000 eth/1 eth/' step-1-deploy
+  # change deploy-testchain not to send 10000 ETH
+  sed_inplace 's/10000 eth/1 eth/' deploy-testchain.sh
 
-  # step 1 only
-  ./step-1-deploy
-
-  MCD_GOV=$(jq -r ".MCD_GOV" "$TDDS/out/addresses.json")
-  MCD_ADM=$(jq -r ".MCD_ADM" "$TDDS/out/addresses.json")
-  ETH_FROM=$(jq -r ".ETH_FROM" "$TDDS/out/addresses.json")
-
-  # set MCD_ADM back to ETH_FROM
-  seth send "$MCD_GOV" 'approve(address,uint256)' "$MCD_ADM" 1
-  seth send "$MCD_GOV" 'mint(uint256)' 1
-  seth send "$MCD_ADM" 'lock(uint256)' 1
-  seth send "$MCD_ADM" 'vote(address[] memory)' ["${ETH_FROM#0x}"]
-  seth send "$MCD_ADM" 'lift(address)' "$ETH_FROM"
-
-  # step 4
-  # jq_inplace '.tokens[].pip.type = "value"' step-4.json
-  # jq_inplace '.tokens.ETH.pip.price = "123.45"' step-4.json
-  # jq_inplace '.tokens.COL1.pip.price = "234.56"' step-4.json
-  # ./step-4-deploy
+  jq_inplace '.tokens[].pip.type = "value"' deploy-testchain.json
+  jq_inplace '.tokens.ETH.pip.price = "123.45"' deploy-testchain.json
+  jq_inplace '.tokens.COL1.pip.price = "234.56"' deploy-testchain.json
```

Comment:
I think the `deploy-testchain.json` has these properties set, or are these exact values expected by the tests?

Reply:
Good call @b-pmcg , I just removed these lines and all the sdk tests pass without them.